### ISSUE TITLE
Explore current status of fine-tuning possibilities

### ISSUE THREAD

**Issue by clessig** [ecmwf/WeatherGenerator#562]
We do not have a good understanding currently about the fine-tuning capabilities of the WeatherGenerator model. Explore these using SYNOP as an example dataset.

**Comment by clessig** [comment]
ERA5 val loss:

<img width="3000" height="2100" alt="Image" src="https://github.com/user-attachments/assets/a45cbb0e-fb49-4b85-ad28-cd701e6767d8" />

Surface val loss:

<img width="3000" height="2100" alt="Image" src="https://github.com/user-attachments/assets/c878d1c5-e30a-4624-82a7-33bc30922704" />

Results are overall as expected. Fine-tuning with fixed latent space is possible also it with limited skill compared to fine-tuning the full model; but this is also only pre-training with 1-deg ERA5.
